doc/development/tutorials/extending_build.rst

```diff
@@ -47,7 +53,8 @@ For that, we will need to add the following elements to Sphinx:
 Prerequisites
 -------------
 
-As with :doc:`helloworld`, we will not be distributing this plugin via PyPI so
+As with :ref:`tutorial-extending-syntax`,
+we will not be distributing this plugin via PyPI so
```

Comment:
Let's update the sample folder structure in this tutorial too, for consistency?

Reply:
No I think that is ok for now

Reply:
I think the files like `todo.py` make sense, because it is an extension to make a todo directive etc, then the actual tutorial is using this as example to demonstrate the actual concepts